skops/io/tests/test_persist.py

```diff
@@ -317,7 +319,10 @@ def get_input(estimator):
     tags = get_tags(estimator)
 
     if tags.input_tags.pairwise:
-        return np.random.rand(N_FEATURES, N_FEATURES), None
+        if not tags.target_tags.required:
+            return np.random.rand(N_FEATURES, N_FEATURES), None
+        else:
+            return np.random.rand(N_FEATURES, N_FEATURES), y[:N_FEATURES]
```

Comment:
Maybe this is ok to be a placeholder before a more thorough refactoring in another PR? It makes the tests pass. Just adding random numbers confuses the estimator if it is classification or regression. I guess we can add a sliced X too?